ls1intum/Artemis
# `Development`: Remove last occurrence of lodash/fp

### Checklist

### Motivation and Context
`lodash/fp` imports make our tests slow, now they are gone and should never be used again!!!

### Steps for Testing

1. Check a couple of pages with data tables, filters and search, e.g. `course management --> students/tutors/editors/instructors` or `student exams` or `course management --> exercise --> participations`


### Review Progress





#### Code Review
#### Manual Tests
